cli/lsp/documents.rs

```diff
@@ -254,6 +262,19 @@ impl AssetOrDocument {
   }
 }
 
+pub fn notebook_specifier(specifier: &Url) -> Option<Url> {
+  if specifier.scheme() == "deno-notebook-cell" {
```

Comment:
It would be a little bit helpful to show an example input and example output in a comment (or perhaps if we added some unit tests for this function).

Reply:
Also renamed to `cell_to_file_specifier()`, the function had changed at some point but I didn't update the name.